jazz-ui/src/app/secondary-components/service-overview-multienv/service-overview-multienv.component.ts

```diff
@@ -10,488 +10,455 @@ import { Subscription } from 'rxjs/Subscription';
-        private authenticationservice: AuthenticationService
-    ) {
-        this.http = request;
-        this.toastmessage = messageservice;
-    }
 
-    email_temp:string;
-    isenvLoading:boolean=false;
-    token:string;
-    noSubEnv:boolean=false;
-    noEnv:boolean=false;
-    slackChannel_temp:string;
-    slackChannel_link:string = '';
-    edit_save:string = 'EDIT';
-    activeEnv:string = 'dev';
-    Environments=[];
-    environ_arr=[];
-    
-    // prodEnv:any;
-    // stgEnv:any;
-    status :string= this.service.status;
-    environments = [
-        {   stageDisp:'PROD',
-            stage : 'prd',
-            serviceHealth : 'NA',
-            lastSuccess : {},
-            lastError : {
-                value:'NA',
-                // unit: 'Days'
-            },
-            deploymentsCount : {
-                'value':'NA',
-                'duration':'Last 24 hours'
-            },
-            cost : {
-                'value': 'NA',
-                'duration': 'Per Day',
-                // 'status': 'good'
-            },
-            codeQuality : {
-                'value': 'NA',
-                // 'status': 'bad'
-            }
-        },
-        {   stageDisp:'STAGE',
-            stage : 'stg',
-            serviceHealth : 'NA',
-            lastSuccess : {
-                value:'NA',
-                // unit: 'Days'
-            },
-            lastError : {},
-            deploymentsCount : {
-                'value':'NA',
-                'duration':'Last 24 hours'
-            },
-            cost : {
-                'value': 'NA',
-                'duration': 'Per Day',
-                // 'status': 'good'
-            },
-            codeQuality : {
-                'value': 'NA',
-                // 'status': 'good'
+
+
+
+  stageClicked(stg) {
+
+    let url = '/services/' + this.service['id'] + '/' + stg
+    this.router.navigateByUrl(url);
+
+  }
+
+
+  toast_pop(error, oops, errorMessage) {
+    var tst = document.getElementById('toast-container');
+    tst.classList.add('toaster-anim');
+    this.toasterService.pop(error, oops, errorMessage);
+    setTimeout(() => {
+      tst.classList.remove('toaster-anim');
+    }, 3000);
+  }
+
+  modifyEnvArr() {
+    var j = 0;
+    var k = 2;
+    this.sortEnvArr();
+
+    if (this.environ_arr != undefined) {
+      for (var i = 0; i < this.environ_arr.length; i++) {
+        this.environ_arr[i].status = this.environ_arr[i].status.replace("_", " ");
+        // this.environ_arr[i].status=this.environ_arr[i].status.split(" ").join("\ n")
+        if (this.environ_arr[i].logical_id == 'prd' || this.environ_arr[i].logical_id == 'prod') {
+          this.prodEnv = this.environ_arr[i];
+          continue;
+        }
+        if (this.environ_arr[i].logical_id == 'stg') {
+          this.stgEnv = this.environ_arr[i];
+          continue;
+        } else {
+          if (this.environ_arr[i].status !== 'archived') {
```

Comment:
Reviewed the UI related changes works fine: approved 
Please get the angular-cli version change and  protractor version change reviewed by surya

Reply:
Done